Source/Core/Core/NetPlayServer.cpp

```diff
@@ -8,7 +8,12 @@
 #include "Common/StdMakeUnique.h"
 #include "Common/StringUtil.h"
 #include "Core/NetPlayServer.h"
+#include "Core/HW/EXI_DeviceIPL.h"
 #include "InputCommon/GCPadStatus.h"
+#include "NetPlayClient.h" //for NetPlayUI
```

Comment:
These need to be relative (just like the other includes surrounding the incorrect ones), and alphabetically placed